sensing_launch/launch/velodyne_VLP16.launch.xml

```diff
@@ -4,101 +4,19 @@
   <arg name="launch_driver" default="true" />
 
   <arg name="model" default="VLP16"/>
-  <arg name="calibration" default="$(find velodyne_pointcloud)/params/VLP16db.yaml"/>
-  <arg name="device_ip" default="192.168.1.201" />
-  <arg name="sensor_frame" default="velodyne" />
-  <arg name="base_frame" default="base_link" />
-  <arg name="manager" default="velodyne_nodelet_manager" />
+  <arg name="calibration" default="$(find-pkg-share velodyne_pointcloud)/params/VLP16db.yaml"/>
   <arg name="max_range" default="130.0" />
   <arg name="min_range" default="0.4" />
-  <arg name="pcap" default="" />
-  <arg name="port" default="2368" />
-  <arg name="read_fast" default="false" />
-  <arg name="read_once" default="false" />
-  <arg name="repeat_delay" default="0.0" />
-  <arg name="rpm" default="600.0" />
-  <arg name="scan_phase" default="0.0" />
-  <arg name="sensor_timestamp" default="false" />
-  <arg name="laserscan_ring" default="-1" />
-  <arg name="laserscan_resolution" default="0.007" />
-  <arg name="num_points_threshold" default="300"/>
   <arg name="invalid_intensity" default="[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]"/>
 
-  <!-- nodelet manager -->
-  <node pkg="nodelet" type="nodelet" name="$(arg manager)" args="manager" />
+  <include file="$(find-pkg-share sensing_launch)/launch/velodyne_node_container.launch.py">
+    <arg name="launch_driver" value="$(var launch_driver)"/>
+    <arg name="model" value="$(var model)"/>
+    <arg name="calibration" value="$(var calibration)"/>
+    <arg name="max_range" value="$(var max_range)"/>
+    <arg name="min_range" value="$(var min_range)"/>
+    <arg name="invalid_intensity" value="$(var invalid_intensity)"/>
```

Comment:
The added argument values should also be passed down to velodyne_node_container.launch.py

```suggestion
    <arg name="invalid_intensity" value="$(var invalid_intensity)"/>
    <arg name="sensor_frame" value="$(var sensor_frame)"/>
    <arg name="device_ip" value="$(var device_ip)"/>
    <arg name="port" value="$(var port)"/>
```

Reply:
done